server/bleep/src/config.rs

```diff
@@ -261,6 +272,29 @@ impl Configuration {
     }
 }
 
+pub fn serialize_secret_opt_str<S>(
+    opt_secstr: &Option<SecretString>,
+    ser: S,
+) -> Result<S::Ok, S::Error>
+where
+    S: Serializer,
+{
+    match opt_secstr {
+        Some(secstr) => ser.serialize_some(secstr.expose_secret()),
+        None => ser.serialize_none(),
+    }
+}
+
+pub fn serialize_secret_str<S>(secstr: &SecretString, ser: S) -> Result<S::Ok, S::Error>
```

Comment:
Does this mean serialization is exfiltrating our secret strings? That sounds just a wee bit scary.

Reply:
In practice we will want to store at least some of these secrets somewhere. I guess dumping github/git secrets may be problematic, but then where do we store them?